Source/WebCore/accessibility/AXCoreObject.cpp

```diff
@@ -566,7 +566,7 @@ void AXCoreObject::appendRadioButtonGroupMembers(AccessibilityChildrenVector& li
             linkedUIElements.append(radioSibling);
     } else {
         // If we didn't find any radio button siblings with the traditional naming, lets search for a radio group role and find its children.
-        for (auto* parent = parentObject(); parent; parent = parent->parentObject()) {
+        for (RefPtr parent = parentObject(); parent; parent = parent->parentObject()) {
             if (parent->roleValue() == AccessibilityRole::RadioGroup) {
                 appendRadioButtonDescendants(*parent, linkedUIElements);
                 break;
```

Comment:
There is an additional `auto*` on line 584. Does that need to be updated too?

Reply:
`layoutContext()` and `isInRenderTreeLayout()` are both inline getters/checks, so I didn't think it met our threshold for converting, but I'll check in on it

Reply:
> @stwrt Abigail is following our smart pointer guidelines, which are well documented and were shared with the rest of the team.

Thanks for the confirmation Chris.

Reply:
Looks like I forgot to put a link: https://github.com/WebKit/WebKit/wiki/Safer-CPP-Guidelines
The beginning of the document is about smart pointers and explains the "trivial functions" aspect.